src/test/java/com/wcc/platform/factories/SetupFactories.java

```diff
@@ -98,6 +106,10 @@ public static Member createCollaboratorsTest() {
     return createCollaboratorMemberTest(MemberType.MEMBER);
   }
 
+  private static PageMetadata createPaginationTest() {
+    return new PageMetadata(new Pagination(1, 1, 1, 10));
```

Comment:
maybe do you want to use this constant here and re-use in CmsServiceIntegrationTest?